index.html

```diff
@@ -1,298 +1,287 @@
 <!DOCTYPE html>
 <html lang="en">
```

Comment:
- [ ] [OPTIONAL] Since you are following the Figma template you may not have control over this. but the color contrast on your page can be improved. If you check it on [this site](https://www.aditus.io/button-contrast-checker/) it currently fails to meet the minimum requirements